src/ssb_project_cli/ssb_project/app.py

```diff
@@ -131,7 +131,7 @@ def make_git_repo_and_push(github_token: str, github_url: str, repo_dir: Path) -
 
     with TempGitRemote(repo, credential_url, username_url):
         repo.git.push("--set-upstream", "origin", "main")
-    print(f"Repo successfully pushed to GitHub: {github_url}")
+    typer.echo(f"Repo successfully pushed to GitHub: {github_url}")
```

Comment:
Why are we changing all of these? The library recommends just using print: https://typer.tiangolo.com/tutorial/printing/#typer-echo